explore/src/main/webapp/agsworker.js

```diff
@@ -0,0 +1,3 @@
+import { AgsWorker } from '@workers/exploreworkers1.js';
```

Comment:
Why `exploreworkers1`?

Reply:
Just testing, all workers should be in the same module now.